### PR TITLE
Sprint5 bookmarks lb - Stories 18 and 24

### PR DESCRIPTION
**Pull request by WorkAccountLB**
includes the constants.js file setup but not implemented

### CODE REVIEW

**Comment by WorkAccountLB** [comment]
There is also a slight change to the "createsubsectionarray.js" component as I had to bring in 1 additional field that had not been implemented previously.

and moved the ContentCCScreen.js PrintTitle component reference into Section.js to facilitate the passing of the bookmark back to the accordion for the CC data.

File: src/assets/icons.js [hunk 1]
```diff
@@ -2,23 +2,66 @@ import React from 'react';
 import Icon from 'react-native-vector-icons/MaterialIcons';
 import styles, {colors} from '../assets/styles';
 
-let iconSize = 20;
+let ICONSIZE = 20; //general programming practice is that all CONSTANTS should be in uppercase*
+let DEVSIZE = 30;
```
Comment (doolaik) [inline src/assets/icons.js:6]: What is in DEVSIZE?

File: src/components/CrimCodeGridList.js [hunk 2]
```diff
@@ -47,12 +47,13 @@ const CrimCodeGridList = props => {
     componentKey = props.heading1Key;
     componentLabel = props.heading1Label;
     componentTitle = props.heading1TitleText;
-  } else {    //SectionCCScreen.js
-    componentKey = props.headingKey;
+  } else {
```
Comment (doolaik) [inline src/components/CrimCodeGridList.js:50]: Eventually the else needs to be removed and make this condition equal to SectionCCScreen etc.

File: src/components/Bookmark.js [hunk 3]
```diff
@@ -1,57 +1,84 @@
...
+        dispatchAction(fullSomeData, localPassKey, localLawType);
       }}>
       <View>
-        <Icon
-          name={marked ? 'bookmark' : 'bookmark-outline'}
-          size={30}
-          style={{color: colors.primary}}
-        />
+        {marked === true && isFocused ? (
+          <BookmarkMarked />
```
Comment (doolaik) [inline src/components/Bookmark.js:78]: Could these names be any more confusing :D

File: src/components/CrimCodeGridList.js (hunk 2 above)
Comment (WorkAccountLB) [reply src/components/CrimCodeGridList.js:50]: I agree.

File: src/components/Bookmark.js (hunk 3 above)
Comment (WorkAccountLB) [reply src/components/Bookmark.js:78]: They TOTALLY make sense.

**Pull request merged**